GameOfRunes/GameOfRunes/Gesture Recogniser/DBPathRecognizer.swift

```diff
@@ -34,8 +34,12 @@ public class GestureRecognizer {
         var path = Path()
         path.addPointFromRaw(rawPoints)
         
+        guard let gameEngine = gameScene.gameEngine else {
```

Comment:
Why do u change from accessing directly to `gameEngine` to accessing to `gameScene.gameEngine`?

Reply:
Trying to better separate the View from the Logic, so all the nodes access the gameEngine from the gameScene.